docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,574 @@
+    -   The `width` key defines the spacing between glyphs. Some commonly-used values include `condensed`, `semiCondensed`, `normal`, `semiExpanded`, and `expanded`.
+    -   The `weight` key defines the thickness or darkness of the glyphs. Some commonly-used values include `thin`, `light`, `normal`, `regular`, `medium`, and `bold`.
+    -   The `slant` key defines whether to use italics. Some commonly-used values include `normal`, `roman`, `italic`, and `oblique`.
+
+        > [!NOTE]
+        > Usually `italic` is a separate font, while `oblique` is generated by applying a slant to the `normal` font.
+
+
+-   `PointSize` is a type float. To calculate the point size from the height in pixels, use the following formula, where `vertical_dpi` is the vertical resolution of a device in dots per inch:
+
+    ```csharp
+    point_size = 72 * pixels / vertical_dpi;
+    ```
+
+The following example code specifies the font properties:
+
+```csharp
+label.FontFamily = "FreeSerif";
+
+PropertyMap fontStyle = new PropertyMap();
+fontStyle.Add("weight", new PropertyValue("bold"));
+fontStyle.Add("slant", new PropertyValue("italic"));
+label.FontStyle = fontStyle;
+label.PointSize = 12.0f;
+```
+
+If no font is specified, default styles are used, and a suitable font for displaying the text label is automatically selected from the platform. However, the automatically-selected font may not render all the characters contained within the text label. For example, Latin fonts often do not provide Arabic glyphs.
+
+### Font Styles
+
+Setting a font size programmatically is not ideal for applications that support multiple screen resolutions, and for platforms that support multiple logical font sizes. In addition, making systemwide changes to your font settings override the font sizes that have been programmatically set.
+
+A more flexible approach is to prepare various JSON stylesheets and request a different style for each platform. The [Tizen.NUI.NUIApplication](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIApplication.html) class has constructors that take a stylesheet argument:
+
+```csharp
+class Example : NUIApplication
+
+Example example = new Example("example-path/example.json");
+```
+
+To change the font style for standard text controls, use the following JSON syntax:
+
+```csharp
+{
+   "styles":
+   {
+      "textlabel":
+      {
+         "fontFamily": "FreeSerif",
+         "fontStyle":
+         {
+            "weight": "bold",
+            "slant": "italic"
+         },
+         "pointSize": 8
+      }
+   }
+}
+```
+
+However, the same `pointSize` is unlikely to be suitable for all text controls in an application. To define custom styles for existing controls, set a style name for each case, and provide a style override in a JSON stylesheet.
+
+You can provide further flexibility for the various screens by mapping the logical size to a physical size in the stylesheet.
+
+<a name="align"></a>
+### Align Text

```

Comment:
@mijin85cho Kindly let me know about your opinion here: 
### Align Text in TextLabel

Reply:
Referencing https://github.com/Samsung/tizen-docs/pull/1009#discussion_r398493508,
I add the updated suggestion

\### Align Text in TextLabel